packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml

```diff
@@ -169,10 +203,15 @@ utm:
       - https://bugzilla.mozilla.org/show_bug.cgi?id=1830504
     expires: never
     data_sensitivity:
-      - web_activity
+      - interaction
 
   source:
-    description: UTM source
+    description: |
+      The source from where the user started.  For example, if the user clicked on a
+      link on the Firefox accounts web site, this value could be 'fx-website'.  The
+      value has a max length of 128 characters with the alphanumeric characters, _
```

Comment:
Drive-by comment: string metrics are limited to 100 characters ([docs](https://mozilla.github.io/glean/book/reference/metrics/string.html)). Everything about that will be truncated. If this is a problem for this data collection we need to know and figure out if we should raise the limit or change the collection.

Reply:
Thanks for pointing that out.  Even though our validation allows up to 128 chars I've never seen any value remotely close to that length, so I think we should be fine.